conu/backend/docker/image.py

```diff
@@ -238,12 +239,16 @@ def _run_container(self, run_command_instance, callback):
             container_id = fd.read()
         return container_id, response
 
-    def run_via_binary(self, run_command_instance=None, *args, **kwargs):
+    def run_via_binary(self, run_command_instance=None, volume=None, *args, **kwargs):
```

Comment:
I propose to name it `volumes` so it's clear it's suppose to be a list

Reply:
Done.